app/coffee/DocumentUpdaterController.coffee

```diff
@@ -12,13 +11,15 @@ MESSAGE_SIZE_LOG_LIMIT = 1024 * 1024 # 1Mb
 module.exports = DocumentUpdaterController =
 	# DocumentUpdaterController is responsible for updates that come via Redis
 	# Pub/Sub from the document updater.
+	rclientList: [redis.createClient(settings.redis.pubsub)]
```

Comment:
were you thinking get the multi client setup live with a single client and then when we are ready todo the migration add `redis.createClient(settings.redis.pubsub)` in a second deploy?

Reply:
yes, this puts the framework in place, then we can add extra entries to the array.

Reply:
if we know what we want, we can add it now.